src/common/reducers/handlers/cards.js

```diff
@@ -116,8 +133,23 @@ function createCardFromProps(props) {
   return card;
 }
 
+function areIdenticalCards(card1, card2) {
+  // TODO: Once we have better UX for this, it's time to start getting stricter
+  // (no longer care about the name, and check abilities/command rather than text).
+  return compareCertainKeys(card1, card2, ['name', 'type', 'cost', 'text', 'stats']);
```

Comment:
compareCertainKeys is not case insensitive which it should be. You don't want duplicate cards with different name capitalization. I also think new capitalization should over-write old capitalization so that you can modify the card names outside of the app and import the new names.